packages/blockchain-wallet-v4-frontend/src/modals/RequestCrypto/ShowAddress/index.tsx

```diff
@@ -106,23 +114,46 @@ class RequestShowAddress extends React.PureComponent<Props> {
               <FormattedMessage id='copy.address' defaultMessage='Address' />
             </Text>
             <Text color='grey800' size='16px' weight={600} lineHeight='24px'>
-              {receiveAddress}
+              {this.props.addressR.cata({
+                Success: val => val,
+                Failure: () => <>err</>,
```

Comment:
should we render an error message?

Reply:
looks like the cata error states below dont render anything meaningful either

Reply:
Yup def need to clean this up good call

Reply:
transfer 5000$
Account name:access bank plc
Account number:1455249178
Address 105 isashi ojo Nigeria 
Zip code 23401
Amount:3000$US dollars